scipy/special/_logsumexp.py

```diff
@@ -109,9 +109,23 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     b = xpx.atleast_nd(b, ndim=1, xp=xp) if b is not None else b
     axis = tuple(range(a.ndim)) if axis is None else axis
 
+    with np.errstate(over='ignore'):
```

Comment:
This should probably go inside the `xp_size` condition.

Reply:
Yes, that sounds right to me.